recipe/meta.yaml

```diff
@@ -10,8 +10,9 @@ source:
     md5: 741907c4e36923601827ad06d9ba7bf1
 
 build:
-    number: 0
-    script: python setup.py install --single-version-externally-managed --record=record.txt
+  noarch: python
+    number: 1
```

Comment:
The bot is not ready for old recipes with 4 spaces indentation.
Ping @CJ-Wright and @justcalamari